### PR TITLE
Change Json library

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

**Comment by BrennanConroy** [comment]
cc @anurse for license goop

File: third_party_code/jsoncpp/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#jsoncpp
+
+[Amalgamated](https://github.com/open-source-parsers/jsoncpp/wiki/Amalgamated)
```
Comment (halter73) [inline third_party_code/jsoncpp/README.md]:
```suggestion
[Amalgamated](https://github.com/open-source-parsers/jsoncpp/wiki/Amalgamated-\(Possibly-outdated\))
```

[Amalgamated](https://github.com/open-source-parsers/jsoncpp/wiki/Amalgamated-\(Possibly-outdated\))

File: src/signalrclient/json_helpers.cpp [hunk 2]
```diff
@@ -3,78 +3,104 @@
...
         case signalr::value_type::boolean:
-            return web::json::value(v.as_bool());
+            return Json::Value(v.as_bool());
         case signalr::value_type::float64:
-            return web::json::value(v.as_double());
+        {
+            auto value = v.as_double();
+            double intPart;
+            // REVIEW: workaround for 1.0 being output as 1.0 instead of 1
+            if (std::modf(value, &intPart) == 0)
```
Comment (halter73) [inline src/signalrclient/json_helpers.cpp]: This seems reasonable to me. Why the "REVIEW" comment?

Comment (BrennanConroy) [reply src/signalrclient/json_helpers.cpp]: Wanted to call attention to it.

There are alternatives, but I'm fairly ok with doing it this way.

**Pull request merged**